content/copilot/building-copilot-extensions/index.md

```diff
@@ -10,11 +10,13 @@ children:
   - /about-building-copilot-extensions
   - /quickstart-for-github-copilot-extensions-using-agents
   - /setting-up-copilot-extensions
+  - /creating-a-copilot-extension
   - /building-a-copilot-agent-for-your-copilot-extension
   - /building-a-copilot-skillset-for-your-copilot-extension
   - /creating-a-copilot-extension
```

Comment:
This line needs to be deleted, it's a duplication left over after merge conflict resolution.